### PR TITLE
Fix initial_state in host and service config. closes #169

### LINKED ISSUE #169: Manage option 'initial_state' on conf (host and service)
initial_state is not managed in alignak.
See https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/objectdefinitions.html

Arguments initial_state don't work because is not present in alignak code but present in the doc.

### PR DESCRIPTION
**Pull request by ddurieux**

### CODE REVIEW

File: alignak/objects/service.py [hunk 1]
```diff
@@ -585,6 +585,18 @@ class Service(SchedulingItem):
...
+        """define state with initial_state
+
+        :return: None
+        """
+        if (self.initial_state == 'w'):
+            self.state = 'WARNING'
+        elif (self.initial_state == 'u'):
+            self.state = 'UNKNOWN'
+        elif (self.initial_state == 'c'):
+            self.state = 'CRITICAL'
```
Comment (Seb-Solon) [inline alignak/objects/service.py]: Hum for hot it's up/down/unreachable
see : https://github.com/Alignak-monitoring/alignak/blob/develop/alignak/objects/host.py#L1151-1160

**Comment by gst** [comment]
Think I'm +1 too for this one.

@Seb-Solon  you allright too ?

**Comment by ddurieux** [comment]
@Seb-Solon it's ok for you?

**Comment by Seb-Solon** [comment]
I though I had already reviewed this. Looks good to me yeah.

**Pull request merged**